rb/.ruby-version

```diff
@@ -1 +1 @@
-3.0.6
+jruby-9.4.5.0
```

Comment:
Makes sense. Makes things a lot simpler and faster for most people, and people who want to look into Ruby can toggle this easily.

Reply:
Actually, I think I committed this by mistake :)
I think we should keep the minimum supported Ruby version here - JRuby 9.4 is a newer version targeting Ruby 3.1.